iced_layershell/src/multi_window.rs

```diff
@@ -683,11 +684,21 @@ async fn run_instance<A, E, C>(
                 }
 
                 let physical_size = window.state.physical_size();
-                compositor.configure_surface(
-                    &mut window.surface,
-                    physical_size.width,
-                    physical_size.height,
-                );
+
+                if match sizes.get(&window.id) {
+                    None => true,
+                    Some(size) => {
+                        size.height != physical_size.height || size.width != physical_size.width
```

Comment:
I think here should also store the scale

Reply:
size can be directly compared I think

Reply:
I think here should also check if it has need to relayout, and this part just check the configure surface